apps/fyle/tasks.py

```
@@ -387,6 +398,390 @@ def update_non_exported_expenses(data: Dict) -> None:
+
+    try:
+        from django_q.models import Schedule
+        from django_q.tasks import schedule
+    except ImportError:
+        logger.warning("Django Q not available, cannot schedule task for workspace %s", workspace_id)
+        return
+
+    logger.info("Scheduling for later processing for changed expense ids %s in workspace %s", changed_expense_ids, workspace_id)
+
+    # generate some random string to avoid duplicate tasks
+    hashed_name = hashlib.md5(str(changed_expense_ids).encode('utf-8')).hexdigest()[0:6]
```

Ruff (0.13.1)

Probable use of insecure hash functions in `hashlib`: `md5`

(S324)